integration_test/Bindings/Python/dialects/rtg.py

```diff
@@ -0,0 +1,69 @@
+# REQUIRES: bindings_python
+# RUN: %PYTHON% %s | FileCheck %s
+
+import circt
+
+from circt.dialects import rtg, rtgtest
```

Comment:
Does this test require a guard for `CIRCT_INCLUDE_TESTS` if it uses rtgtest?

Reply:
The whole `integration_test` folder is already guarded by `CIRCT_INCLUDE_TESTS`